db/v-2/tables/project.xml

```diff
@@ -13,4 +13,17 @@
         </addColumn>
     </changeSet>
 
+    <changeSet id="project-add-user" author="petertrr" context="dev or prod">
+        <addColumn tableName="project">
+            <column name="user_id" type="bigint">
+<!-- Shouldn't be nullable, but mappings should be done at first -->
+                <constraints foreignKeyName="fk_project_user" references="user(id)" nullable="true"/>
+            </column>
+        </addColumn>
+        <addColumn tableName="project">
+            <column name="admin_ids" type="varchar(256)">
```

Comment:
what about superusers? where do you plan to store them?

Reply:
These will be users with `ROLE_ADMIN` in authorities. USers, that are project admins, will have something like `ROLE_PROJECT_ADMIN`